test/cql-pytest/test_bloom_filter.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+# Test inserts `N` rows into table, flushes it 
+# and tries to read `M` non-existing keys.
+# Then bloom filter's false-positive ratio is checked.
+
+import pytest
+import random
+import nodetool
+from util import new_test_table
+
+def test_bloom_filter(cql, test_keyspace):
+    N = 10000 # Number of writes
+    M = 5000  # Number of reads
```

Comment:
perhaps it's worth parametrizing (N,M) to test a couple of these pairs? Ref: https://docs.pytest.org/en/6.2.x/parametrize.html

Reply:
Oh wait, I actually meant parametrizing `fp` as well.